config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -727,6 +727,7 @@ periodics:
   annotations:
     testgrid-dashboards: sig-scalability-benchmarks
     testgrid-tab-name: scheduler-perf
+    testgrid-alert-email: patrick.ohly@intel.com
```

Comment:
Thanks @pohly I think it makes sense to add kubernetes-sig-scheduling@googlegroups.com as well (copied kubernetes-sig-scale@googlegroups.com above).

Reply:
Added to both.